clients/admin-ui/src/features/system/EditSystemFlow.tsx

```diff
@@ -4,16 +4,15 @@ import { useRouter } from "next/router";
 import { useCallback, useState } from "react";
```

Comment:
Once we no longer support nav v1, this can likely be consolidated with `SystemFormTabs.tsx`